### PR TITLE
fix(config): update Keemple Smart Radiator

### PR DESCRIPTION
**Pull request by zollek**
Discription, product type and Wake Up command update.

PR description here

### CODE REVIEW

File: packages/config/config/devices/0x0002/keemple_smart_radiator.json [hunk 1]
```diff
@@ -12,5 +16,21 @@
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "132",
+			"label": "Wake Up",
+			"description": "Wake Up Interval",
+			"valueSize": 60,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/keemple_smart_radiator.json:25]: That's not a valid value size, must be 1...4 bytes.
Also, please add a unit.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0002/keemple_smart_radiator.json [hunk 2]
```diff
@@ -12,5 +16,21 @@
...
+			"minValue": 60,
+			"maxValue": 1800,
+			"defaultValue": 300
+		}
+	],
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/keemple_smart_radiator.json]: This doesn't seem correct. 132 is the ID of the `Wake Up CC`, which has nothing to do with the configuration parameters that are defined through `paramInformation`.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x0002/keemple_smart_radiator.json (hunk 2 above)
Comment (zollek) [reply packages/config/config/devices/0x0002/keemple_smart_radiator.json]: I've hoped that updating this command I can change Wake Up Interval in my battery thermostat.
I don't have any z-wave device that has parameters to configure, so I didn't realize I can't update Wake Up time it this way.
Deprecated OpenZWave integration had an option to set it through UI, do You know how can I change it in Z-Wave JS?

**Review by AlCalzone — APPROVED** [review]

File: packages/config/config/devices/0x0002/keemple_smart_radiator.json (hunk 2 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0002/keemple_smart_radiator.json]: See [my other comment](https://github.com/zwave-js/node-zwave-js/pull/3807#discussion_r759589266)

**Pull request merged**